### PR TITLE
update spring boot version

### PR DESCRIPTION
**Pull request by indiepopart**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL approved by Dev Advocacy
- [ ] The content has been run through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist
- [ ] Images are compressed appropriately
- [ ] Social image previews well on Twitter and LinkedIn
- [ ] Tech review request (developer advocate or domain expert)
- [ ] Review request for editorial/grammar/clarity (developer advocate or Colton)

### CODE REVIEW

File: _source/_posts/2024-01-15-spring-boot-graphql-react.md [hunk 1]
```diff
@@ -701,11 +699,10 @@ Download the following seed files to an empty directory, as it will be mounted t
 - [LandOwnershipAmericans.csv](https://guides.neo4j.com/ukcompanies/data/LandOwnershipAmericans.csv)
 - [PSCAmericans.csv](https://guides.neo4j.com/ukcompanies/data/PSCAmericans.csv)
 
-Create the directory `src/main/docker` and create a file `neo4j.yml` there, with the following content:
+Spring Boot’s Docker Compose integration now supports Neo4j. Edit the file `compose.yml` and add a service for the Neo4j database.
```
Comment (mraible) [inline _source/_posts/2024-01-15-spring-boot-graphql-react.md]:
```suggestion
Spring Boot's Docker Compose integration now supports Neo4j. Edit the `compose.yml` file and add a service for the Neo4j database.
```

File: _source/_posts/2024-01-15-spring-boot-graphql-react.md [hunk 2]
```diff
@@ -1284,6 +1268,8 @@ Get an access token using the Auth0 CLI with the `auth0 test token` command:
 auth0 test token -a https://<your-auth0-domain>/api/v2/
```
Comment (mraible) [inline _source/_posts/2024-01-15-spring-boot-graphql-react.md]:
```suggestion
auth0 test token -a https://<your-auth0-domain>/api/v2/ -s openid
```

File: _source/_posts/2024-01-15-spring-boot-graphql-react.md [hunk 3]
```diff
@@ -1284,6 +1268,8 @@ Get an access token using the Auth0 CLI with the `auth0 test token` command:
...
 ```
 
+Select the **CLI Login Testing** app or any available client when prompted, you don't need to select any scope. You will also be prompted to open a browser window and log in with a user credential.
```
Comment (mraible) [inline _source/_posts/2024-01-15-spring-boot-graphql-react.md:1271]: I think it's better to add `-s openid` as a scope so the user isn't confused when they're prompted for one.

**Comment by indiepopart** [comment]
Hi @mraible! Applied suggestions and more minor edits.

**Review by mraible — APPROVED** [review]

**Pull request merged**